modules/oadp-kopia-configuring-algorithms.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/overriding-kopia-algorithms.adoc          
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-kopia-configuring-algorithms_{context}"]
+= Configuring the DPA to override Kopia hashing, encryption, and splitter algorithms
+
+{oadp-short} provides you with the option to override the default Kopia algorithms for hashing, encryption, and splitter to improve Kopia performance or to compare performance metrics. You can set the following environment variables in the `spec.configuration.velero.podConfig.env` section of the DPA:
+
+* KOPIA_HASHING_ALGORITHM
+* KOPIA_ENCRYPTION_ALGORITHM
+* KOPIA_SPLITTER_ALGORITHM
+
+.Prerequisites
+
+* You have installed the {oadp-short} Operator.
+* You have created the secret by using the credentials provided by the cloud provider.
+* You have installed the Kopia client.
```

Comment:
That is not really required for selecting algorithm, the kopia CLI is only required for connecting and benchmarking, however see my comments around using must-gather.

Reply:
Removed this line. Have added steps for configuring a mustgather pod and running kopia benchmarking commands via ssh.